Source/WebCore/inspector/agents/WebConsoleAgent.cpp

```diff
@@ -45,6 +45,16 @@ namespace WebCore {
 
 using namespace Inspector;
 
+#if USE(APPLE_INTERNAL_SDK) && __has_include(<WebKitAdditions/WebConsoleAgentAdditions.cpp>)
+#include <WebKitAdditions/WebConsoleAgentAdditions.cpp>
+#else
+static String networkConnectionIntegrityErrorMessage(const ResourceError& error)
+{
+    UNUSED_PARAM(error);
```

Comment:
NIT: you could just have it be `(const ResourceError&)` and not need the `UNUSED_PARAM` :)